Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -1919,6 +1961,46 @@ inline StyleSelfAlignmentData BuilderConverter::convertSelfOrDefaultAlignmentDat
         }
     } else
         alignmentData.setPosition(fromCSSValue<ItemPosition>(value));
+
+    // Flip the position according to position-try fallback, if specified.
+    if (auto positionTryFallback = builderState.positionTryFallback()) {
```

Comment:
We might want to factor applying positionTryFallback applying to a separate class (AnchorPositionEvaluator or a new one) as the amount of code related to it in the converter is getting pretty large.

Reply:
I suppose this would be the last (or close to the last) addition to fallback handling, so we could refactor later. Tracked here https://bugs.webkit.org/show_bug.cgi?id=291111